### ISSUE TITLE
Anvils can be lifted with telekinesis and thrown no matter where they are

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#33419]
(WEB REPORT BY: theophrastus REMOTE: 206.221.180.138:7777)
# Revision
0a6bf586f84cca1b145e82451b6c808be61cedcb

# Description
Anvils can be brought into hand sprite with telekinesis while they still exist on the ground and thrown no matter how far away you are from the anvil.

# Steps to Reproduce
Get telekinesis
get anvil
acme the station
# What you Expected
to not have an anvil thrown at me

# What Actually Happened
i got owned

**Comment by Theophrastus1** [comment]
This issue should most likely be closed since I made a mistake in identifying the Force from the sith bundle and mutation as Telekinesis.

**Comment by Theophrastus1** [comment]
I will point out that this is the Sith Bundle version of TK, The Force that is doing this

**Comment by nervere** [comment]
@Theophrastus1 Any other details you can share about how you picked up the anvil? I gave myself the telekinesis from the Sith bundle (`/obj/item/weapon/dnainjector/nofail/telemut`) and can't seem to replicate this. More specifically, I can't pick up the anvil from a distance whatsoever. Nothing happens.

**Comment by Theophrastus1** [comment]
> @Theophrastus1 Any other details you can share about how you picked up the anvil? I gave myself the telekinesis from the Sith bundle (`/obj/item/weapon/dnainjector/nofail/telemut`) and can't seem to replicate this. More specifically, I can't pick up the anvil from a distance whatsoever. Nothing happens.

I witnessed it in a round, I myself didn't use it. According to Razz on the discord, apparently what he thinks is happening is that the anvil is being picked up by the bound item, since anvils are considered items. You CAN pick anvils up but it requires hulk as well as strong to do so. Apparently bound item is bypassing that check, and allowing you to bring the anvil to hand slot, then throw the anvil.

A few logs that I dredged up showing it occurring, but there isn't much logging on it. It is displayed in the attack log however of him attacking with only having the advanced tk and jumpy, but no hulk or anything.

[13:36:34]ATTACK: Syd Robinson (spacedgod) used the DNA-Injector (Tele.) to inject Syd Robinson (spacedgod)
[13:36:41]ATTACK: Syd Robinson (spacedgod) used the DNA-Injector (Jumpy) to inject Syd Robinson (spacedgod)

[13:42:16]ATTACK: <font color='red'>Syd Robinson(spacedgod) attacked Hingle McCringleberry(silent3) with anvil. (INTENT: HELP) (DAMTYE: BRUTE)</font>
[13:42:16]ATTACK: <font color='red'>Syd Robinson(spacedgod) damaged Hingle McCringleberry(silent3) with the anvil. DMG: 40</font>
[13:42:24]ATTACK: <font color='red'>Syd Robinson(spacedgod) attacked Hingle McCringleberry(silent3) with anvil. (INTENT: DISARM) (DAMTYE: BRUTE)</font>
[13:42:24]ATTACK: <font color='red'>Syd Robinson(spacedgod) damaged Hingle McCringleberry(silent3) with the anvil. DMG: 40</font>
[13:42:25]ATTACK: <font color='red'>Syd Robinson(spacedgod) attacked Hingle McCringleberry(silent3) with anvil. (INTENT: DISARM) (DAMTYE: BRUTE)</font>
[13:42:25]ATTACK: <font color='red'>Syd Robinson(spacedgod) damaged Hingle McCringleberry(silent3) with the anvil. DMG: 40</font>

[13:51:50]ATTACK: <font color='red'>Brecht Breelholm (theophrastus) was hit by the anvil thrown by Syd Robinson (spacedgod)

[14:04:41]ATTACK: <font color='red'>Landen Blackburn (dtscurvy) was hit by the anvil thrown by Syd Robinson (spacedgod)</font>

![image](https://user-images.githubusercontent.com/8057687/193499964-d834ba36-8851-4b7a-bf05-b4568ba42fed.png)

**Comment by kacurasuo** [comment]
Yes, this has to do with bound object and not telekinesis. Binding an item teleports it to your hands, including anvils.

**Comment by kacurasuo** [comment]
forget throwing the anvil, just use it as a 40 damage melee weapon (that can also be thrown for 40 damage)
